pkg/windows/windows.go

```diff
@@ -1008,6 +1014,21 @@ func (vm *windows) removeHNSNetwork(networkName string) error {
 	return nil
 }
 
+// enableWindowsFeatures enables the required Windows Container features on the Windows host.
+func (vm *windows) enableWindowsFeatures() error {
```

Comment:
Consider renaming to `enableRequiredWindowsFeature`

Reply:
I would leave it plural (`...WindowsFeatures()`), so that it can be extended with other required features.

Reply:
I'd keep it singular - it can be changed once we have more

Reply:
I lean towards keeping it singular as well in the short term.